File: ws/Perez/U2/WS24/Zoo/src/main/java/ec/edu/espe/zoo/model/Tiger.java
package ec.edu.espe.zoo.model;

import java.util.Date;

/**
 *
 * @author Carlos Perez,The Javas, DCCO-ESPE
 */
public class Tiger extends Feline{

    public Tiger(int id, String description, Date bornOnDate, char gender, Cage cage, int numberOfBones, boolean vetebrate) {
        super(id, description, bornOnDate, gender, cage, numberOfBones, vetebrate);
    }

    @Override
    public void feed(int food) {
        System.out.println("feeding the lion with meat");
    }
    
}

File: hw/Pérez/U2/HW18ZooJavaScript/model/Tiger.js
class Tiger extends Feline {
    constructor(id, description, bornOnDate, gender, cage, numberOfBones, vetebrate) {
        super(id, description, bornOnDate, gender, cage, numberOfBones, vetebrate);
    }

    feed(food) {
        console.log("Feeding the tiger with meat");
    }
}
